classes/DirtSoundLibrary.sc

```diff
@@ -179,7 +203,9 @@ DirtSoundLibrary {
 		};
 
 		files = pathMatch(folderPath.standardizePath +/+ "*"); // dependent on operating system
-		if(sortFiles) { files.sort };
+		if(sortFiles) { 
+			 files.sort;	
```

Comment:
in sclang, there is no `;` inside the last statement of a function. It is not wrong, but the change is not needed.